### PR TITLE
Fixes #420 #421 :- Improves Media Wall

### LINKED ISSUE #420: Adding header to loklak media wall
Add a header to loklak media wall.

### PR DESCRIPTION
**Pull request by Achint08**
**Changes proposed in this pull request**
- Added Hover effects and actions to media wall cards
- Added header to media wall
- Made media wall card more responsive.
- Some minor changes to feed card.

**Screenshots (if appropriate)** 

screen shot 2017-06-24 at 12 27 18 am

**Link to live demo (if appropriate):** 
https://achint08.github.io/loklak_search/

**Closes #420 #421**
Please review @hemantjadon @djmgit @kavithaenair @vibhcool @singhpratyush @rishiraj824 .

### CODE REVIEW

**Comment by singhpratyush** [comment]
Adding to what @hemantjadon said, would the transition look better if it were a little slower?

**Comment by singhpratyush** [comment]
Here is a screenshot from Firefox - 
screenshot from 2017-06-23 11 28 10

Also, due to the explicit `40px` margin, it looks weird on small screens (iPhone 6) - 
achint08 github io-loklak_search-wall-query a filter video iphone 6 1


I'm not sure why it shows 3 cards even on small devices. Maybe the grid system can be helpful for this purpose.

The problem will also be there for large screens. If the display is too wide (51" for example), the content will be there at the extreme positions on the screen (as the margin is just `40px`) and user would have trouble reading it. I would recommend using the Bootstrap `container` for this. Please let me know your views on this.

And please inform me if these changes need to go in separate issues (which I think they should).

**Comment by Achint08** [comment]
@singhpratyush
> Here is a screenshot from Firefox -

 Is it looking like this on firefox? Not for me. 
@hemantjadon Any idea?

**Comment by Achint08** [comment]
@hemantjadon @singhpratyush Checkout now. :)

**Comment by hemantjadon** [comment]
@Achint08 Why z-index?? Please read on the internet about this. Also please make sure UI looks decent, IMHO This is not mergeable, The shadows are too dark, try some sites, see how they handle such effects try to implement the same.

**Comment by Achint08** [comment]
@hemantjadon I previously, used z-index to create an effect but removed it due to overlapping of cards. Checkout now. Was unable to deploy at that time. :(

**Comment by Achint08** [comment]
@hemantjadon @singhpratyush Please review so that I can improve this or proceed to next step.

**Comment by singhpratyush** [comment]
@Achint08 @hemantjadon: Here are some points. Please share your views - 
1. Viewport usage can be improved by using `container`. It would make the website look better on a variety of displays.
2. No link underlines.
3. Elevation: `2px 2px 2px` to `7px 7px 7px` on hover.
4. Not using `vw` as `font-size` unit.

Here is a screenshot after changes mentioned in points 1, 2 and 3 - 

screenshot from 2017-06-24 14 13 36

**Comment by Achint08** [comment]
@singhpratyush 
> Viewport usage can be improved by using container. It would make the website look better on a variety of displays.

screen shot 2017-06-24 at 2 19 46 pm


It should not look like this. Like i have used wrapper to display it on 100% of the screen. tested on http://quirktools.com/ for all types of screens.

> No link underlines.

Will look that in the later PR.

> Not using vw as font-size unit.

For larger screens to adjust font size according to view port. This is the reason I used this.

**Comment by singhpratyush** [comment]
> i have used wrapper to display it on 100% of the screen.

That's what I was asking to change if valid. Full-width views are not good for viewing purpose if the centre content is as important as the boundary content (Tweets in this case). You should keep all important things within the unstretched viewing area of the user.

**Comment by Achint08** [comment]
@singhpratyush  Done! 👍

**Comment by hemantjadon** [comment]
@Achint08, Not looking good on small displays. It's getting too small. Please check.

**Comment by Achint08** [comment]
@hemantjadon Are you talking about the columns or the elements?

**Comment by hemantjadon** [comment]
@Achint08 The text and the elements, try resizing the screen, you will notice

**Comment by Achint08** [comment]
@hemantjadon Done! Checkout now.
http://achint08.github.io/loklak_search

File: src/app/media-wall/media-wall.component.scss [hunk 1]
```diff
@@ -45,16 +37,31 @@ body {
     -webkit-box-sizing: border-box;
     box-shadow: 2px 2px 4px 0 #ccc;
 		overflow: auto;
```
Comment (hemantjadon) [inline src/app/media-wall/media-wall.component.scss]: Indentation error

**Comment by Achint08** [comment]
Done! @hemantjadon.

**Pull request merged**